src/main/java/pl/company/foodatu/meals/domain/InMemoryMealPublisher.java

```diff
@@ -0,0 +1,25 @@
+package pl.company.foodatu.meals.domain;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.company.foodatu.meals.dto.MealResponse;
+import pl.company.foodatu.meals.infrastructure.MealPublisher;
+
+import java.util.Queue;
+import java.util.concurrent.ArrayBlockingQueue;
+
+class InMemoryMealPublisher implements MealPublisher {
+
+    private static final Logger logger = LoggerFactory.getLogger(InMemoryMealPublisher.class);
```

Comment:
We can use this logger to log when new meal is published on the queue. Can be helpful when debugging tests